setup.py

```diff
@@ -49,6 +50,10 @@
 # Preference for PyQt5 if you select ALL...
 extras_require['all'].remove('PySide')
 
+entry_style = "gui_scripts"
+if platform.system() == "Windows":
```

Comment:
Had to double-check, but this looks good: `Returns the system/OS name, e.g. 'Linux', 'Windows' or 'Java'.`

Reply:
Yup... better than the `sys.platform` which returns `win32` and it always puzzles me when they are going to switch it to `win64` and break all software.